### ISSUE TITLE
Unable to give aliases to or hide network interfaces and disks

### ISSUE THREAD

**Issue by jcz14** [nicolargo/glances#1126]
#### Description
I am unable to give aliases to or hide network interfaces using the global configuration file due to their names being truncated in the sidebar. In addition, I am unable to give aliases to network interfaces with spaces in their name.
![glances-sidebar](https://user-images.githubusercontent.com/14078374/28486824-7018a8aa-6e3b-11e7-851e-62dc16e032c2.png)
#### Versions
* Glances: 2.10
* PSutil: 5.2.2
* Operating System: Windows Server 2008 R2

**Comment by nicolargo** [comment]
Have you try to backslash the space or use the \s.
Glances use regular expression: https://stackoverflow.com/questions/559363/matching-a-space-in-regex

**Comment by jcz14** [comment]
I was able to get the full name of the network interfaces and disks through PSutil. Using `\s` in place of spaces allows me to match network interfaces to be hidden but does not allow me to give them aliases. For example `Ethernet\s1_alias=Eth1` does not take effect.

**Comment by nicolargo** [comment]
@jcz14 In Glances configuration file regexp are only available for the value. The following configuration should work (hide Ethernet 2 and alias Ethernet 1):

    [network]
    Ethernet 1_alias=Eth1
    hide=Ethernet\s2

Perhaps the problem cam from the case sensitive.

Please can you start Glances in debug mode (-d) and send us back [the log file](http://glances.readthedocs.io/en/stable/config.html#logging) via [pastebin](https://pastebin.com/).

**Comment by jcz14** [comment]
Adding the directives you provided doesn't hide Ethernet 2 or alias Ethernet 1. [Here](https://pastebin.com/pKZ1YbY5) is the log file.

**Comment by nicolargo** [comment]
Ok reproduce and solve in the dev branch !

Can you test it ?

**Comment by jcz14** [comment]
The disk alias takes effect but network interface aliasing still does not work. In addition, network interface hiding stopped working when I installed the scripts under the dev branch. No changes were made to the configuration file.

**Comment by nicolargo** [comment]
@jcz14 Can you copy the contain of your Glances conf file ?

**Comment by jcz14** [comment]
[Here it is.](https://pastebin.com/ty0wjRHD)

**Comment by nicolargo** [comment]
@jcz14 Can you try this for the network configuration:

```
[network]
# Define the list of hidden network interfaces (comma-separated regexp)
hide=isatap.{EC143621-4695-4CB9-9EF5-F88E41E2B075},isatap.{6927214B-DF01-4A19-B3AF-7FB4D449260B},isatap.{BA5F5D3D-EBC0-4CDF-9B23-D8C217AED629},isatap.{4D608786-C969-4805-98B1-7060B5BFB5F8},isatap.{AA77CA2C-EE70-4928-8FC0-DF6F69487A18},Teredo Tunneling Pseudo-Interface,Loopback Pseudo-Interface 1
Ethernet 1_alias=eth0
Ethernet 2_alias=eth1
Ethernet 3_alias=eth2
Ethernet 4_alias=eth3
IMM2-40f2e9d28ea4_alias=rsa
```

**Comment by jcz14** [comment]
Using the above for network configuration does not produce the desired result. The hidden interfaces are still hidden but the ethernet adapters are still not aliased.

**Comment by nicolargo** [comment]
Can you try this ?

```
[network]
ethernet 1_alias=eth0
ethernet 2_alias=eth1
ethernet 3_alias=eth2
ethernet 4_alias=eth3
```

**Comment by atriwidada** [comment]
I got the same bug. Can't hide interfaces like veth0UEVHH from lxc with explicit hide=veth0UEVHH in [network], but it works (can be hidden) if I put hide=veth* instead. glances v2.11.1

**Comment by nicolargo** [comment]
@atriwidada It should be solved in the develop branch (future Glances 3.0).
